src/index.js

```diff
@@ -17,7 +17,32 @@ try {
   console.error(error);
 }
 
-const client = new Client({ intents: [GatewayIntentBits.Guilds] });
+const client = new Client({
+  intents: [
+    GatewayIntentBits.Guilds,
+    GatewayIntentBits.GuildMessages,
+    GatewayIntentBits.MessageContent,
+  ],
+});
+const prefix = '&'; // 봇 명령어 접두사
+
+client.on('messageCreate', (message) => {
```

Comment:
해당 부분은 Events를 사용해서 리팩토링 하는게 좋겠네요
```suggestion
client.on(Events.MessageCreate, (message) => {
```